doc/user/content/self-managed/installation/install-on-aws.md

```diff
@@ -0,0 +1,362 @@
+graceful degradation rather than OOMing. *Network-attached* storage (like EBS
+volumes) can significantly degrade performance and is not supported.
+
+For locally-attached NVMe storage, we recommend using OpenEBS with LVM Local PV
+for managing local volumes. While other storage solutions may work, we have
+tested and recommend OpenEBS for optimal performance.
+
+For locally-attached NVMe storage, install OpenEBS to your running Kubernetes
+cluster.
+
+```bash
+# Add OpenEBS to Helm
+helm repo add openebs https://openebs.github.io/openebs
+helm repo update
+
+# Install only the Local PV Storage Engines
+helm install openebs --namespace openebs openebs/openebs \
+  --set engines.replicated.mayastor.enabled=false \
+  --create-namespace
+```
+
+Verify the installation:
+```bash
+kubectl get pods -n openebs -l role=openebs-lvm
+```
+
+#### Logical Volume Manager (LVM) configuration
+
+Logical Volume Manager (LVM) setup varies by environment. Below is our tested
+and recommended configuration:
+
+##### AWS EC2 with Bottlerocket AMI
+
+Tested configurations:
+
+|                                              |      |
+|----------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Instance types**                           | **r6g**, **r7g** families <br> **Note:** LVM setup may work on other instance types with local storage (like i3.xlarge, i4i.xlarge, r5d.xlarge), but we have not extensively tested these configurations. |
+| **AMI**                                      | AWS Bottlerocket |
+| **Instance store volumes**                   | Required |
+
+To setup:
+
+1. Use Bottlerocket bootstrap container for LVM configuration.
+1. Configure volume group name as `instance-store-vg`
+
+{{< tip >}}
+
+If you are using the recommended Bottlerocket AMI with the Terraform module,
+the LVM configuration is automatically handled by the EKS module using the
+provided user data script.
+
+{{< /tip >}}
+
+To verify the LVM setup, run the following:
+
+```bash
+kubectl debug -it node/<node-name> --image=amazonlinux:2
+chroot /host
+lvs
+```
+
+You should see a volume group named `instance-store-vg`.
+
+## 1. Install the Materialize Operator
+
+1. If installing for the first time, create a namespace. The default
+   configuration uses the `materialize` namespace.
+
+   ```bash
+   kubectl create namespace materialize
+   ```
+
+1. Create a `my-AWS-values.yaml` configuration file for the Materialize
+   operator. Update with details from your AWS Kubernetes environment. For more
+   information on cloud provider configuration, see the [Materialize Operator
+   Configuration](/self-managed/configuration/#operator-parameters).
+
+      ```yaml
+      # my-AWS-values.yaml
+      # Note: Updated with recent config changes in main branch and not v0.125.2 branch
```

Comment:
Wasn't sure what configs we're planning to use when people sign up. So, I just went with what's recently in main.